dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -278,8 +278,8 @@ func generateTarball(skipUI, skipHelm bool) error {
 
 	mvnArgs := getCommonMvnArgs(hadoopVersion)
 	if skipUI {
-		mvnArgsNoUI := append(mvnArgs, "-pl", "!webui")
-		run("compiling repo without UI", "mvn", mvnArgsNoUI...)
+		mvnArgs = append(mvnArgs, "-pl", "!webui")
+		run("compiling repo without UI", "mvn", mvnArgs...)
```

Comment:
// Compile ufs modules for the main build
	buildModules(srcPath, "underfs", ufsModulesFlag, version, ufsModules, mvnArgs)
Never use mvnArgsNoUI... that arguments only used for logging?